_posts/2018-01-02-twitter-cards-for-jekyll-with-jekyll-seo-tag.md

```diff
@@ -34,6 +34,13 @@ This will make GitHub pages know to use the plugin.
 
 To install locally run `gem install jekyll-seo-tag` at the command line.
 
+Finally, add the following tag inside the `<head>` in your site's template(s):
+
+```
+{% raw %}{% seo %}{% endraw %}
+```
+```
```

Comment:
Looks like you have an extra closing code fence now.

Reply:
Sorry, fixed.